### PR TITLE
feat(skymp5-server): implement an imitation of bound weapons

### LINKED ISSUE #1279: Implement an imitation of bound weapons
We have spell books that summon some weapons. When we consume a such one, we have to add a player a conjured weapon. Thus, we simulate magick spell.

### PR DESCRIPTION
**Pull request by che6a**
Closes #1279.

### CODE REVIEW

File: skymp5-server/cpp/server_guest_lib/PieScript.cpp [hunk 1]
```diff
@@ -432,10 +446,30 @@ void PieScript::Play(MpActor& actor, const WorldState& worldState,
...
+      actor.GetChangeForm().actorValues.magickaPercentage;
+    if (currentMagickaPercentage >= it->second.GetManacostPercentage()) {
+      actor.DamageActorValue(espm::ActorValue::Magicka,
+                             it->second.GetManacost());
+      uint32_t boundWeaponBaseId = it->second.GetBaseId(),
+               bookBaseId = it->first;
+      actor.AddItem(boundWeaponBaseId, 1);
+      actor.RemoveItem(bookBaseId, 1, nullptr);
+      worldState.SetTimer(it->second.GetCooldown())
+        .Then([bookBaseId, boundWeaponBaseId, &actor](Viet::Void) {
```
Comment (Pospelove) [inline skymp5-server/cpp/server_guest_lib/PieScript.cpp:467]: &actor can be destroyed at this point. so instead of & we should use formId and =

Comment (Pospelove) [reply skymp5-server/cpp/server_guest_lib/PieScript.cpp:467]: + added items should be also equipped

**Review by Wedmer — APPROVED** [review]

File: skymp5-server/cpp/server_guest_lib/MpObjectReference.h [hunk 2]
```diff
@@ -124,6 +124,7 @@ class MpObjectReference
   void SetInventory(const Inventory& inv);
   void AddItem(uint32_t baseId, uint32_t count);
   void AddItems(const std::vector<Inventory::Entry>& entries);
+  void ForceEquip(uint32_t baseId);
```
Comment (Pospelove) [inline skymp5-server/cpp/server_guest_lib/MpObjectReference.h:127]: please remove this method and replace with sending a packet to a client to equip something

File: skymp5-server/cpp/server_guest_lib/SweetPieBoundWeapon.cpp [hunk 3]
```diff
@@ -0,0 +1,42 @@
+#include "SweetPieBoundWeapon.h"
```
Comment (Pospelove) [inline skymp5-server/cpp/server_guest_lib/SweetPieBoundWeapon.cpp:1]: do not add a separate file for this OR
at least name files the same way (for example Not SwwetPie*** and PieScript, but
SweetPie******* and SweetPieScript)

**Review by Pospelove — CHANGES_REQUESTED** [review]

File: skymp5-server/cpp/server_guest_lib/Inventory.cpp [hunk 4]
```diff
@@ -199,3 +199,13 @@ Inventory Inventory::FromJson(const nlohmann::json& j)
   simdjson::dom::element parsed = p.parse(j.dump());
   return FromJson(parsed);
 }
+
+void Inventory::SetWorn(uint32_t baseId, Worn worn)
```
Comment (Pospelove) [inline skymp5-server/cpp/server_guest_lib/Inventory.cpp:203]: this method isnt needed then

**Review by Pospelove — CHANGES_REQUESTED** [review]
please also do not change vcpkg version in this pr

**Comment by che6a** [comment]
Sure, I will fix this when I get back home.

**Review by Pospelove — CHANGES_REQUESTED** [review]
now vcpkg is deleted from the repo. it should stay the same

**Review by Pospelove — APPROVED** [review]

**Comment by che6a** [comment]
@Pospelove ready fore review

**Pull request merged**